make/dev.mk

```diff
@@ -11,7 +11,7 @@ REGISTRY_URI=quay.io
 DEVCONSOLE_OPERATOR_IMAGE?=quay.io/redhat-developers/devconsole-operator
 TIMESTAMP:=$(shell date +%s)
 TAG?=$(GIT_COMMIT_ID_SHORT)-$(TIMESTAMP)
-OPENSHIFT_VERSION?=3
+OPENSHIFT_VERSION?=4
 OC_LOGIN_USERNAME?=system:admin
```

Comment:
@baijum we can remove `OC_LOGIN_USERNAME` and `OC_LOGIN_PASSWORD` since we are no longer using it.

Reply:
Good point :-)

Reply:
Done d3e2600adb44d28f32b3010d70a9a7580df1260b